sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
Mmm... don't know why that was there in the first place, but thanks! That would be good!